### DISCUSSION TITLE
[Feature Request]:

### DISCUSSION THREAD

**Discussion by prennings** [beyondcode/herd-community#370] [Ideas]
### Feature Description

Hi Guys,

Currently the docs have a section for adding extra extensions to Laravel Herd. But as soon as you need extra extensions for multiple versions of php you are going to brew hell pretty fast.

I found out that most extensions can be downloaded from brew pre-compiled;
https://github.com/shivammathur/homebrew-extensions

For simplicity it maybe smart to change the documentation regarding the extra extensions to use above brew install method and edit the php.ini file pointing to the installed .so downloaded by brew;

As example to download sqlsrv.so for php 8;
`brew install shivammathur/extensions/pdo_sqlsrv@8.0`

After installing brew will say where the files are located as following;
`/opt/homebrew/Cellar/pdo_sqlsrv@8.0/5.11.1`

There the sqlsrv.so file is stored.

The only thing left is to edit the correct php.ini version and add something like;
`extension=/opt/homebrew/Cellar/sqlsrv@8.0/5.11.1/sqlsrv.so`

Works great for me and much easier.

**Comment by mpociot** [comment]
Oh nice, that's indeed easier than having to install them manually via pecl.
I'll add it to the docs - thank you for bringing this up! 👍
